guides/common/modules/proc_benchmarking-raw-db-performance.adoc

```diff
@@ -6,7 +6,7 @@ To get a list of the top table sizes in disk space for both Candlepin, Foreman,
 endif::[]
 
 PGbench utility (note you may need to resize PostgreSQL data directory `{postgresql-lib-dir}` directory to 100 GiB or what does benchmark take to run) might be used to measure PostgreSQL performance on your system.
-Use `{package-install} postgresql-contrib` to install it.
+Use `{client-package-install} postgresql-contrib` to install it.
```

Comment:
@ekohl This also looks like it should be `project-`. Can you confirm?

Reply:
That's correct.

Now that I'm reading this I'm looking at what this chapter is supposed to achieve. It's quite. Katello-centric in assuming that Candlepin and Pulp are present.

Overall I wonder what the point of the linked sizes script is. https://stackoverflow.com/questions/18907047/postgres-db-size-command says you can use `\l+` the psql utility and that does work for me:
```console
$ sudo -u postgres psql
psql (12.12)
Type "help" for help.

postgres=# \l+
                                                                    List of databases
   Name    |  Owner   | Encoding |   Collate   |    Ctype    |   Access privileges   |  Size   | Tablespace |                Description                 
-----------+----------+----------+-------------+-------------+-----------------------+---------+------------+--------------------------------------------
 foreman   | foreman  | UTF8     | en_US.utf8  | en_US.utf8  | foreman=CTc/foreman  +| 38 MB   | pg_default | 
           |          |          |             |             | =T/foreman            |         |            | 
 postgres  | postgres | UTF8     | nl_NL.UTF-8 | nl_NL.UTF-8 |                       | 7337 kB | pg_default | default administrative connection database
 template0 | postgres | UTF8     | nl_NL.UTF-8 | nl_NL.UTF-8 | =c/postgres          +| 8089 kB | pg_default | unmodifiable empty database
           |          |          |             |             | postgres=CTc/postgres |         |            | 
 template1 | postgres | UTF8     | nl_NL.UTF-8 | nl_NL.UTF-8 | postgres=CTc/postgres+| 7177 kB | pg_default | default template for new databases
           |          |          |             |             | =c/postgres           |         |            | 
(4 rows)
```

Perhaps @evgeni knows better if there's a difference.

While on the topic: since Pulp 3.22 (included in Katello 4.9) we always require postgresql-contrib anyway, but it's still needed on vanilla Foreman (without Katello).

Then about the specific PGbench part: I'm not sure what's expected of the reader.

In conclusion: what's the point of this chapter?

Reply:
Well, the linked script shows table sizes, not database sizes, so the output is quite different.
Whether this amount of detail is required at that point or what the reader should do with it, I do not know.

(I also prefer not to link to random GitHub repos in docs, but that's another totally different topic).

Reply:
> Well, the linked script shows table sizes, not database sizes (there is `\dt+` for table sizes, but that's still less detail than the script), so the output is quite different.

Ah, good point.

> Whether this amount of detail is required at that point or what the reader should do with it, I do not know.

I also wonder what the point of this chapter is. Yes, you know table sizes. But then what do you do with it?

> (I also prefer not to link to random GitHub repos in docs, but that's another totally different topic).

Also what triggered me and why I started to look at native PostgreSQL capabilities. So you can inline it.